src/Microsoft.ManagedZlib/tests/MyClassTests.cs

```diff
@@ -0,0 +1,339 @@
+    {
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.doc") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.docx") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.pdf") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.txt") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "alice29.txt") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "asyoulik.txt") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "cp.html") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "fields.c") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "grammar.lsp") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "kennedy.xls") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "lcet10.txt") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "plrabn12.txt") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "ptt5") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "sum") };
+        //yield return new object[] { Path.Combine("UncompressedTestFiles", "xargs.1") };
+    }
+
+    //[Theory]
+    //[MemberData(nameof(UncompressedTestFiles))]
+    //public async Task Read(string testFile)
+    //{
+    //    LocalMemoryStream uncompressedStream = await LocalMemoryStream.readAppFileAsync(testFile);
+    //    LocalMemoryStream compressedStream = await LocalMemoryStream.readAppFileAsync(CompressedTestFile(testFile));
+    //    using Stream decompressor = CreateStream(compressedStream, CompressionMode.Decompress);
+    //    var decompressorOutput = new MemoryStream();
+
+    //    int _bufferSize = 1024;
+    //    var bytes = new byte[_bufferSize];
+    //    bool finished = false;
+    //    int retCount;
+    //    while (!finished)
+    //    {
+    //        retCount = decompressor.Read(bytes, 0, _bufferSize);
+
+    //        if (retCount != 0)
+    //            decompressorOutput.Write(bytes, 0, retCount);
+    //        else
+    //            finished = true;
+    //    }
+    //    decompressor.Dispose();
+    //    decompressorOutput.Position = 0;
+    //    uncompressedStream.Position = 0;
+
+    //    byte[] uncompressedStreamBytes = uncompressedStream.ToArray();
+    //    byte[] decompressorOutputBytes = decompressorOutput.ToArray();
+
+    //    Assert.Equal(uncompressedStreamBytes.Length, decompressorOutputBytes.Length);
+    //    for (int i = 0; i < uncompressedStreamBytes.Length; i++)
+    //    {
+    //        Assert.Equal(uncompressedStreamBytes[i], decompressorOutputBytes[i]);
+    //    }
+    //}
+    [Fact]
+    public void Test1()
```

Comment:
Consider renaming the tests to something meaningful that explains the purpose of the test. Here and elsewhere.